tests/integration/test_api.py

```diff
@@ -422,30 +422,36 @@ def test_annotate_insight_anonymous_then_authenticated(client):
     assert insight.items() > {"n_votes": 1}.items()
 
     # then the user connects and vote for same insights
-    authenticated_result = client.simulate_post(
-        "/api/v1/insights/annotate",
-        params={
-            "insight_id": insight_id,
-            "annotation": 1,
-            "device_id": "voter1",
-        },
-        headers={"Authorization": "Basic " + base64.b64encode(b"a:b").decode("ascii")},
-    )
 
-    assert authenticated_result.status_code == 200
-    assert authenticated_result.json == {
-        "description": "the annotation was saved",
-        "status": "saved",
-    }
-    # We have the previous vote, but the last request should validate the insight directly
-    votes = list(AnnotationVote.select())
-    assert len(votes) == 1  # this is the previous vote
+    if insight["process_after"] != None:
```

Comment:
`make checks` fails here with the message: 

`./tests/integration/test_api.py:426:33: E711 comparison to None should be 'if cond is not None:'`

Reply:
But with this, test passes just because insight["process_after"] is None ! So the rest of the code is skipped !
Why did you put that. 
It should instead check that process_after is empty.
You can do this by adding it to assert at line 421:

```python
assert not any(insight[key] for key in ("username", "completed_at", "annotation", "process_after"))
```